commands/core/config.drush.inc

```diff
@@ -444,10 +444,10 @@ function _drush_config_export($destination, $destination_dir, $branch) {
 
     $config_comparer = new StorageComparer($comparison_source, $target_storage, Drupal::service('config.manager'));
     if (!$config_comparer->createChangelist()->hasChanges()) {
-      return drush_log(dt('There are no changes to export.'), LogLevel::OK);
+      return drush_log(dt('The active config does not differ to the export directory (!target).', array('!target' => $destination_dir)), LogLevel::OK);
     }
 
-    drush_print("The following configuration changes have been made since the last export:\n");
+    drush_print("Differences of the active config to the export directory:\n");
```

Comment:
See: https://github.com/drush-ops/drush/commit/f7eccff3cac6fe1ef643a496d1ec0f06070f8a4d#commitcomment-15784936
the big Problem I see that the changes are not between the last export and the current export (would imply that we're using the snapchot system), but it is just between the active config and the export directory.